### PR TITLE
adjust article template and card block

### PR DESCRIPTION
**Pull request by rrusher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix the single card display on an article.

Test URLs:
- Before: https://main--24life--hlxsites.hlx.page/snack-attack
- After: https://single-card--24life--hlxsites.hlx.page/fuel/2020/snack-attack

### CODE REVIEW

File: blocks/card/card.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ export default function decorate(block) {
       { width: '300' },
     ],
   );
+  // adjust style if only 1 card in the block
```
Comment (synox) [inline blocks/card/card.js:36]:
```suggestion
  // adjust style if only 1 card in the section
```

File: blocks/card/card.js [hunk 2]
```diff
@@ -33,6 +33,10 @@ export default function decorate(block) {
...
+  if (block.children.length === 1) {
+    block.closest('.card-container').classList.add('single');
+  }
```
Comment (synox) [inline blocks/card/card.js:39]: I suggest to remove this, and instead on https://github.com/hlxsites/24life/blob/19c1581d70ccde979cb566db743f929d033a1276/blocks/card/card.css#L62-L70 make default behavior to be `display:block`, and then add a `three-columns` (and 4 if you are already at it). 
Like this we don't have multiple ways to configure the columns of a block, and a reasonable default behavior.

File: templates/article/article.js [hunk 3]
```diff
@@ -4,12 +4,15 @@ export default async function decorate(doc) {
...
+  // remove h1 and picture
+  doc.querySelector('main .section .default-content-wrapper h1').remove();
+  doc.querySelector('main .section .default-content-wrapper p').remove();
   const firstContent = doc.querySelector('main .section .default-content-wrapper');
   firstContent.before(createSocialMediaButtons());
 
-  const lastContent = doc.querySelector('main .section:last-child .default-content-wrapper');
-  lastContent.after(createAuthorBlock());
-  lastContent.after(createSocialMediaButtons());
+  const lastContent = doc.querySelector('main .section:last-child');
```
Comment (synox) [inline templates/article/article.js:13]: see as well merge conflict
```suggestion
  const lastContent = [...doc.querySelectorAll('main .section .default-content-wrapper')].at(-1);
```

File: templates/article/article.js [hunk 4]
```diff
@@ -4,12 +4,15 @@ export default async function decorate(doc) {
   const firstSection = doc.querySelector('main .section');
   firstSection.before(createSectionWithHeroBlock());
 
+  // remove h1 and picture
+  doc.querySelector('main .section .default-content-wrapper h1').remove();
+  doc.querySelector('main .section .default-content-wrapper p').remove();
```
Comment (synox) [inline templates/article/article.js:9]: I already remove them here https://github.com/hlxsites/24life/blob/72aa01a1c82a2c17fcf82580b60ff0cb8466fc74/templates/article/article.js#L37-L39

```suggestion
```

**Review by synox — CHANGES_REQUESTED** [review]

**Comment by synox** [comment]
@rrusher can you please clean this up and merge, as merge conflicts are accumulating and https://github.com/hlxsites/24life/pull/68 depends on this one.

File: blocks/card/card.js (hunk 2 above)
Comment (rrusher) [reply blocks/card/card.js]: I switched to columns block. http://localhost:3000/drafts/rrusher/card-columns

**Review by synox — APPROVED** [review]
nice, much simpler 👍

**Pull request merged**